lib/core/src/client/manager/provider.js

```diff
@@ -32,7 +32,7 @@ export default class ReactProvider extends Provider {
     }
 
     const queryString = qs.stringify(queryParams);
-    const url = `iframe.html?${queryString}`;
+    const url = `preview.html?${queryString}`;
```

Comment:
Can't we make it backward compatible somehow? I like the name "preview" much more than "iframe", however, there are a lot of tools (like visual testing) that are accessing directly the iframe.html

Reply:
I think we should actively update `storyshots` and inform other things that rely on `iframe.html` if we're making this change.

Reply:
@igor-dv I'll try it and ask for feedback. Could be that there will be pressure to keep it as is, and I might revert in that case.

We won't know unless we try & ask.

If you have good suggestions on backward compatibility whilst pressuring people to move to the new location (so we don't have to keep supporting both forever) I'd be happy to hear your suggestions?!

Reply:
@tmeasday ☝️

Reply:
@oblador 👆